add-on/src/popup/browser-action/mdc.switch.css

```diff
@@ -0,0 +1,172 @@
+/*!
```

Comment:
No module available for `mdc.switch.css` ?

Reply:
`@material/switch` - but it only allows you to override the "on" state:

> It is not currently possible to customize the color of a disabled or off (unchecked) switch
https://www.npmjs.com/package/@material/switch#sass-mixins

Our "off" state is the same as our "on" state (visually), we're not so much on/off as left/right so I needed to edit the CSS a little to achieve this.